routes/index.js

```diff
@@ -96,6 +94,19 @@ exports.conversation = function (req, res) {
 	});
 }
 
+exports.account = function (req, res){
+	if (!req.user || !req.user._id) {
+		res.render('error', {
+			title: "Looks like you're not logged in!"
+		});
+		return;
+	} else{
+		res.render('account', {
+			user: req.user
+		});
+	}
+}
```

Comment:
Yeah, the exports.account function handles requests to the "/account" page, so if the user is not logged in, they don't have access, so once we know that `req.user == null`, we can just render the error page with a custom title.  Otherwise, if the user is logged in, we can render the account page for them.

Hm, I realize that alot of the functions I wrote don't have comments.  I'll be sure to add some here and there to make it clear what the handlers do.